src/core/database/database.service.ts

```diff
@@ -71,6 +71,40 @@ export const permission = (property: string, baseNode: string) => {
   ];
 };
 
+export const permissions = (baseNode: string, properties: string[]) => {
+  return properties.flatMap((property) => permission(property, baseNode));
+};
+
+export const property = (
+  prop: string,
+  value: any | null,
+  baseNode: string,
+  propVar = prop,
+  propLabel?: string
+) => {
+  const createdAt = DateTime.local().toISO();
+  // let propLabel = 'Property';
+  // if (prop === 'position') {
+  //   propLabel = 'Property:InternPosition';
+  // } else if (prop === 'methodologies') {
+  //   propLabel = 'Property:ProductMethodology';
+  // } else if (prop === 'status') {
+  //   propLabel = 'Property:EngagementStatus';
+  // }
+  return [
+    [
+      node(baseNode),
+      relation('out', '', prop, {
+        active: true,
+        createdAt,
+      }),
+      node(propVar, propLabel ?? 'Property', {
```

Comment:
`Property` could be added here regardless.
```suggestion
      node(propVar, ['Property', propLabel], {
```